### PR TITLE
[Settings - Users]: Adding form validation messages

### LINKED ISSUE #373: [Settings - User]: Editing user , accepts invalid email address
Steps to Reproduce: 
1. Login to the application
2. Click on Settings
3. Click on User
4. Add new User
5. Click to edit the new user 
6. Enter invalid email address . For example : gerilife440@gmail
7. Notice, allows to save changes with incorrect email address

Video : https://www.loom.com/share/9e1b74c9c58641ce88bdf317da2b9d75

### LINKED ISSUE #374: [Settings-User]: Creating user with invalid data doesn't display error/validation message
Steps to Reproduce
1. Login to the application
2. Click on Settings
3. Click on User
4. Click on Add User
5. Enter invalid data
6. Notice, no validation message is displayed
Screenshot : http://prntscr.com/njfxt9
Video : https://www.loom.com/share/213076efdf004cd3b203e613d6c552c2

### LINKED ISSUE #375: [Settings-Enroll user]: Validation message should be displayed for incorrect data or no data entered
Steps to Reproduce
1. Login to the application
2. Click on Settings
3. Click on Users
4. Click on Send invite(s) button without entering any data
5. Notice, no validation message is displayed

Video : https://www.loom.com/share/213076efdf004cd3b203e613d6c552c2

### PR DESCRIPTION
**Pull request by shailee-m**
closes #373 , closes #375 , closes #374 

Adding correct email validation regex

Adding flash messages individually for all forms in settings/users

### CODE REVIEW

File: app/client/layouts/settings/settings.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     {{> settingsLayoutHeader }}
     {{> settingsLayoutSidebar }}
     <div class="content-wrapper">
-      {{> flashMessages}}
+      {{> flashMessages}}
```
Comment (brylie) [inline app/client/layouts/settings/settings.html]: This seems like lint. Clean up trailing whitespace.

File: app/client/views/settings/users/edit/edit.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template name="editUser">
   <div class="modal fade">
     <div class="modal-dialog">
-      <div class="modal-content">
+      <div class="modal-content">
```
Comment (brylie) [inline app/client/views/settings/users/edit/edit.html]: Again, clean up trailing whitespace.

**Comment by brylie** [comment]
This PR looks ready to merge, other than the lint. Please clean up lint, and we can merge it.

**Comment by shailee-m** [comment]
> This PR looks ready to merge, other than the lint. Please clean up lint, and we can merge it.

Done!

**Pull request merged**